test/utils/runFixtureTests.js

```diff
@@ -3,10 +3,14 @@ var getFixtures = require("babel-helper-fixtures").multiple;
 
 exports.runFixtureTests = function runFixtureTests(fixturesPath, parseFunction) {
   var fixtures = getFixtures(fixturesPath);
+  var testGrep = process.env.TEST_GREP || "";
 
   Object.keys(fixtures).forEach(function (name) {
     fixtures[name].forEach(function (testSuite) {
       testSuite.tests.forEach(function (task) {
+        if (task.actual.loc.indexOf(testGrep) === -1) {
```

Comment:
Since this defaults to `""`, won't this always be `0`? "thing".indexOf("") === 0`